specs/rearrangements.yaml

```diff
@@ -139,6 +147,10 @@ fields:
     type: integer
     doc: FWR4 end coord in sequence (transferred from germline)
   -
+    name: duplicate_count
+    type: integer
+    doc: Count of duplicate sequences for this sequence
```

Comment:
This docstring seems unclear to me.  (Actually, I'm not sure I understand what this field is)